modules/gdscript/gdscript.cpp

```diff
@@ -2019,6 +2019,55 @@ Error GDScriptLanguage::execute_file(const String &p_path) {
 }
 
 void GDScriptLanguage::finish() {
+	finalize();
+}
+
+void GDScriptLanguage::finalize() {
+	if (finalized) {
+		return;
+	}
+	finalizing = true;
+
+	if (_call_stack) {
+		memdelete_arr(_call_stack);
+	}
+
+	// Clear the cache before parsing the script_list
+	GDScriptCache::clear();
+
+	// Clear dependencies between scripts, to ensure cyclic references are broken
+	// (to avoid leaks at exit).
+	SelfList<GDScript> *s = script_list.first();
+	while (s) {
+		// This ensures the current script is not released before we can check
+		// what's the next one in the list (we can't get the next upfront because we
+		// don't know if the reference breaking will cause it -or any other after
+		// it, for that matter- to be released so the next one is not the same as
+		// before).
+		Ref<GDScript> scr = s->self();
+		if (scr.is_valid()) {
+			for (KeyValue<StringName, GDScriptFunction *> &E :
+					scr->member_functions) {
```

Comment:
The wrapping doesn't seem to really make the code more readable here, I would keep it on a single line (same below).

Reply:
I had problems with my gitrepo, and in the process, it reformatted everything (weirdly).
I'll fix this.